macOS-HS-libvirt.xml

```diff
@@ -66,18 +61,18 @@
     <feature policy='require' name='xsave'/>
     <feature policy='require' name='avx'/>
     <feature policy='require' name='xsaveopt'/>
-    <feature policy='require' name='avx2'/>
     <feature policy='require' name='smep'/>
+    <!-- Note: Remove the next line when your CPU does not support AVX2 -->
+    <feature policy='require' name='avx2'/> 
   </cpu>
-<!-- Note: To resolve clock issues experienced on Fedora 27 as described at top of file,
-     we need to add a timer. Replace the clock offset line with,
-
-     <clock offset='utc'>
-       <timer name='tsc'/>
-     </clock>
-
-     Please remember to also change the kvm hidden state as well, as described above.
--->
+  <clock offset='utc'>
+  <!-- Note: The frequency is required for KVMs "Suspend to Disk" feature and
+       must match your host CPUs TSC frequency (in Hz).
+       You can obtain the TSC frequency under Linux with "dmesg | grep -i tsc".
+       When you don't need "Suspend to Disk" delete the frequency attribute.
+  -->
+    <timer name='tsc' frequency='3400022000'/>
```

Comment:
Can you please experiment with `-cpu Penryn,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on` options?

Once these options are implemented, no manual tsc tweaking should be required.

Reply:
The `kvm=on` may be required when exposing NVIDIA GPUs. I am not 100% sure about this.

Reply:
does that timer frequency feature add allow snapshots to work?

Reply:
> Once these options are implemented, no manual tsc tweaking should be required.

@Ghabry Hi, did these options help?